Dockerfile_clearlinux

```diff
@@ -1,15 +1,109 @@
+FROM clearlinux as build
+# Stage for building TF serving API for Python 3.8
+RUN swupd bundle-add \
+    c-basic \
+    curl \
+    devpkg-curl \
+    git \
+    devpkg-libusb \
+    deprecated-python2 \
+    python3-basic \
+    wget \
+    patch  \
+    diffutils \
+    dev-utils \
+    unzip \
+    which
+# Set up Bazel
+ENV BAZEL_VERSION 0.24.1
+WORKDIR /bazel
+RUN curl -H "User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36" -fSsL -O https://github.com/bazelbuild/bazel/releases/download/$BAZEL_VERSION/bazel-$BAZEL_VERSION-installer-linux-x86_64.sh && \
+    curl -H "User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36" -fSsL -o /bazel/LICENSE.txt https://raw.githubusercontent.com/bazelbuild/bazel/master/LICENSE && \
+    chmod +x bazel-*.sh && \
+    ./bazel-$BAZEL_VERSION-installer-linux-x86_64.sh && \
+    cd / && \
+    rm -f /bazel/bazel-$BAZEL_VERSION-installer-linux-x86_64.sh
+
+RUN git clone -b 2.0.0 https://github.com/tensorflow/serving
+WORKDIR serving
+RUN pip3 install wheel
+RUN mkdir -vp /.cache/bazel/
+RUN bazel build --color=yes --curses=yes \
+    --verbose_failures \
+    tensorflow_serving/tools/pip_package:build_pip_package
+RUN bazel-bin/tensorflow_serving/tools/pip_package/build_pip_package /tmp/pip
+
+RUN swupd repair -V 32750 --force
+
+# Build OpenVINO and nGraph (OV dependency) with D_GLIBCXX_USE_CXX11_ABI=0
+RUN git clone --recurse-submodules -j4 https://github.com/opencv/dldt --branch 2020.3.0 --single-branch --depth 1 /openvino
```

Comment:
Perhaps we could go with new url here? -  https://github.com/openvinotoolkit/openvino